src/components/AccountCard.js

```diff
@@ -25,27 +25,34 @@ import AccountIcon from './AccountIcon';
 import TouchableItem from './TouchableItem';
 
 export default class AccountCard extends React.PureComponent<{
-  title: ?string,
   address: string,
   chainId: string,
-  onPress: () => any
+  onPress: () => any,
+  title: ?string,
+  seedType?: string,
 }> {
   static propTypes = {
-    title: PropTypes.string,
     address: PropTypes.string.isRequired,
-    style: ViewPropTypes.style,
     chainId: PropTypes.string,
-    onPress: PropTypes.func
+    onPress: PropTypes.func,
+    seedType: PropTypes.string,
+    style: ViewPropTypes.style,
+    title: PropTypes.string,
   };
 
   static defaultProps = {
     title: 'no name'
   };
 
   render() {
-    const { address, chainId, style, onPress } = this.props;
+    const { address, chainId, onPress, seedType, style } = this.props;
     let { title } = this.props;
     title = title.length ? title : AccountCard.defaultProps.title;
+    let seedTypeDisplay = ''
+
+    if (seedType && seedType!== ''){
```

Comment:
`seedType !== ''`

Reply:
if `seedType` is undefined `seedType!== ''` will be true. Since `seedType` is an optional `prop` I should check that it's defined isn't it?